examples/build_overrides/ti_simplelink_sdk.gni

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+declare_args() {
+  # Root directory for TI SimpleLink SDK build helpers.
+  ti_simplelink_sdk_build_root =
+      "//third_party/connectedhomeip/third_party/ti_simplelink_sdk"
+
+  # Root of the TI SimpleLink SDK for TI devices
+  # can be overriden by environment variable TI_SIMPLELINK_SDK_ROOT @ref ${ti_simplelink_sdk_build_root}/ti_simplelink_board.gni
+  ti_simplelink_sdk_cc13x2_26x2_root =
+      getenv("HOME") + "/ti/simplelink_cc13x2_26x2_sdk_4_30_00_54"
+  freertos_root = getenv("HOME") + "/third_party/FreeRTOSv10.3.1"
```

Comment:
This should be in its own file, unless it's a fork and you want to call it `ti_freertos_root`

Reply:
Moved it out to a git submodule.